core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1021,6 +1025,94 @@ write_files:
       rkt rm --uuid-file=/var/run/coreos/decrypt-assets.uuid || :
 {{ end }}
 
+{{if .Experimental.NodeDrainer.Enabled}}
+  - path: /srv/kubernetes/manifests/kube-node-drainer-asg-ds.yaml
+    content: |
+        kind: DaemonSet
+        apiVersion: extensions/v1beta1
+        metadata:
+          name: kube-node-drainer-sg-ds
+          namespace: kube-system
+          labels:
+            k8s-app: kube-node-drainer-sg-ds
+        spec:
+          updateStrategy:
+            type: RollingUpdate
+          template:
+            metadata:
+              labels:
+                k8s-app: kube-node-drainer-sg-ds
+              annotations:
+                scheduler.alpha.kubernetes.io/critical-pod: ''
+            spec:
+              tolerations:
+              - operator: Exists
+                effect: NoSchedule
+              - operator: Exists
+                effect: NoExecute
+              - operator: Exists
+                key: CriticalAddonsOnly
+              initContainers:
+                - name: hyperkube
+                  image: {{.HyperkubeImage.RepoWithTag}}
+                  command:
+                  - /bin/cp
+                  - -f
+                  - /hyperkube
+                  - /workdir/hyperkube
+                  volumeMounts:
+                  - mountPath: /workdir
+                    name: workdir
+              containers:
+                - name: main
+                  image: {{.AWSCliImage.RepoWithTag}}
+                  env:
+                  - name: NODE_NAME
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: spec.nodeName
+                  command:
+                  - /bin/sh
+                  - -xec
+                  - |
+                    kubectl()  { /lib/ld-musl-x86_64.so.1 /opt/bin/hyperkube kubectl "$@"; }
```

Comment:
instead of publishing new image, we copy `hyperkube` binary from hyperkube image into awscli image. Hyperkube image is based on GNU Libc and awscli is based on musl.  `hyperkube` is not a static binary, therefore it requires ELF interpreter, which when build with GNU Libc is `/lib64/ld-linux-x86-64.so.2`. Under the hood when you run `./hyperkube` what actually running is `/lib64/ld-linux-s86-64.so2 ./hyperkube`, binary is given as an argument to interpreter. 

awscli image is based on Musl and therefore misses `/lib/lid-linux-x86-64.so.2` interpreter, it provides another one `/lib/ld-musl-x86_64.so.1` which we use here to load `hyperkube` binary